### DISCUSSION TITLE
Observable External Asset

### DISCUSSION THREAD

**Discussion by kornerc** [dagster-io/dagster#23729] [Q&A] [answered]
With Dagster 1.8 `external assets` are marked stable and `source assets` deprecated.
In my pipelines I use the decorator `observable_source_asset` to get [observable source assets](https://docs.dagster.io/concepts/assets/asset-observations#observable-source-assets).

Since source assets are deprecated and will get removed in version 2.0.0, is there a similar decorator to `observable_source_asset` for external assets?

**Comment by sryza** [comment, marked as answer]
Hey @kornerc - we are currently working on this. Here's one of the proposals: https://github.com/dagster-io/dagster/pull/23405. For now, I'd recommend sticking with the `@observable_source_asset` decorator.
